src/main/java/org/opensearch/timeseries/indices/IndexManagement.java

```diff
@@ -11,6 +11,7 @@
 
 package org.opensearch.timeseries.indices;
 
+import static org.opensearch.ad.settings.AnomalyDetectorSettings.FLATTENED_ANOMALY_RESULTS_INDEX_MAPPING_FILE;
```

Comment:
try not use ad variable in a time series package. It is meant for both AD and forecasting.